### DISCUSSION TITLE
Hide the menubar on second display

### DISCUSSION THREAD

**Discussion by umtiagopereira** [FelixKratz/SketchyBar#417] [Q&A]
Hi!

I'm a noob, so I don't know how to code. Sorry if this was already answered before; **I tried to search but didn't understand much the solutions proposed.**

Basically, what I want to do it's to keep the macos menubar exactly how it is but hide it on my second display (in this case, it's the built-in display).

I couldn't figure out a way to do this on macos settings other than making all the displays in the same space, but this disable some features like the dock on the second display and being able to go full screen on the first screen while not on the second.

So I thought that maybe using a customization app to replace the menubar could solve this, and then I found SketchyBar.

So, it's possible?

**Comment by GKestenberg** [comment]
In short: the `display` field should be set to "main" if you want that kind of functionality. In your sketchybarrc file add a flag: `sketchybar --bar display=main ...`. Hope that helps for anyone reading up on this issue.

For more info you can read up on https://felixkratz.github.io/SketchyBar/config/bar.
